tests/ftests/ftests.sh

```diff
@@ -4,6 +4,14 @@
 START_DIR=$PWD
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 
+# synchronize between different github runners running on
+# same VM's, this will stop runners from stomping over
+# each other's run.
+LIBCGROUP_RUN_DIR="/var/run/libcgroup/"
+RUNNER_LOCK_FILE="/var/run/libcgroup/github-runner.lock"
+RUNNER_SLEEP_SECS=600		# sleep for 10 minutes
```

Comment:
10 minutes feels like a long wait time to me, but I haven't thought about it much :/

Reply:
An average run takes between 10 to 15 minutes to complete, hence the lower bound of 10 minutes.